### PR TITLE
Easily and correctly allow the user to install requirements

### PR DESCRIPTION
**Pull request by Bluenix2**
This simplifies and unifies how users will install our requirements

### CODE REVIEW

File: dev-requirements.txt [hunk 1]
```diff
@@ -3,9 +3,6 @@
 # This list will have to be migrated if you wish to use another dependency manager.
 
 # Base tools
-pillow~=8.3.1
-numpy~=1.21.0
-blessed~=1.18.1
```
Comment (robincunningham2) [inline dev-requirements.txt]: All our requirements have to be in `dev-requirements.txt` I think

> \# This file contains all the development requirements for our linting toolchain.
> \# Don't forget to pin your dependencies!

Comment (Bluenix2) [reply dev-requirements.txt]: Yes, but we also have `requirements.txt` that the user will install when using our app. No need for them to install our development requirements.

Comment (robincunningham2) [reply dev-requirements.txt]: When our project gets reviewed, they will use `dev-requirements.txt` I think. Let's just keep them both

Comment (Tobi-De) [reply dev-requirements.txt]: leave them in both then, we also need those packages in dev anyway

Comment (Bluenix2) [reply dev-requirements.txt]: Yeah I fixed that now

**Review by Tobi-De — APPROVED** [review]

**Pull request merged**